mitiq/tests/test_utils.py

```diff
@@ -31,17 +31,22 @@
     S,
     T,
     MeasurementGate,
+    ops,
     depolarize,
 )
 
 from mitiq.utils import (
+    _append_measurements,
     _are_close_dict,
     _equal,
+    _is_measurement,
+    find_all_qubits_with_terminal_measurements,
```

Comment:
Note: I moved some utility functions in `zne.scaling` because I wanted to use them for the 1st implementation of something in `Observable`. The current implementation does not use them, but it's fine / better to move these into utils.